Packs/EDL/Integrations/EDL/EDL.yml

```diff
@@ -63,6 +63,13 @@ configuration:
   name: url_port_stripping
   required: false
   type: 8
+- additionalinfo: If selected, a slash will be appended to URLs composed only of an hostname.
+    For example, 'www.example.com' would become 'www.example.com/'
+  defaultvalue: 'false'
```

Comment:
@DeanArbel default is now "false". This should work, right?

Reply:
yeah, this should be alright. The default`False` should handle `None` cases for instances saved before the change